scripts/core-suite-deployments/1_reference_mainnet_flagship_V3Core_deployer.ts

```diff
@@ -13,7 +13,7 @@ import { MinterSetPriceERC20V2__factory } from "../contracts/factories/MinterSet
 import { MinterDALinV2__factory } from "../contracts/factories/MinterDALinV2__factory";
```

Comment:
FWIW, I don't think you have to go back and update all of these reference files. might be nice to add a single deployer script for adding this minter to existing core deployements (see `./scripts/minter-deployments/`), but most of these scripts are kept for archival purposes/starting points for future deployments. We should formalize what our entire script retention policy is too, just haven't prioritized that 👍

Reply:
Sounds good, ok I went ahead and added the deployment scripts for dev & mainnet